src/include/kompute/Tensor.hpp

```diff
@@ -341,59 +346,3 @@ class TensorT : public Tensor
 };
 
 } // End namespace kp
-
-/**
- * fmt fromater for kp::Tensor::TensorDataTypes.
- */
-template<>
-struct fmt::formatter<kp::Tensor::TensorDataTypes> : formatter<std::string>
```

Comment:
Why are fmt converters removed? Would still be required when spdlog not a dependency

Reply:
No, not with the version of fmt we are using.